### ISSUE TITLE
Open  PDF and scoop up text to feed into ChatGPT

### ISSUE THREAD

**Issue by cmosguy** [hunkimForks/chatgpt-arxiv-extension#17]
Is there a way to just read in the PDF itself and get the text for prompting?  Doing an *abstract* in itself can only go so far.  I want to be able to interact with the paper and all the contents within it.

I noticed that there are other files in zips, like HTML, etc if you go to the Other Formats link: https://arxiv.org/format/1607.00148v2

Can we somehow crawl that data to feed the prompt?

**Comment by hunkim** [comment]
Yes, it is possible to use embedding. Please look at this example: https://github.com/openai/openai-cookbook/blob/main/apps/web-crawl-q-and-a/web-qa.py

Feel free to send us PR. :-)

**Comment by QuantumWitness** [comment]
For papers that include the TeX source, it's probably easier to redirect to [ar5iv.org](https://ar5iv.labs.arxiv.org/) or [arxiv-vanity.org](https://www.arxiv-vanity.com/).
